auth-web/src/components/auth/stepper/InfoStepper.vue

```diff
@@ -2,24 +2,26 @@
   <div>
     <v-container id="step-buttons-container">
       <template v-for="(step, index) in steps">
-        <div class="step" :key="index">
+        <div class="step" :key="index"  @click="$router.push(step.to)">
           <v-btn
             fab
             outlined
             class="step__icon"
             active-class="filled no-active"
+            :ripple="false"
             :id=step.id
             color="#1A5A96"
-            :to=step.to>
+            :to="step.to">
             <v-icon>{{step.step}}</v-icon>
           </v-btn>
-          <v-btn class="step__label" active-class="selected no-active" text color="#1A5A96" :to=step.to>
+          <v-btn class="step__label" :ripple="false" active-class="selected no-active" text color="#1A5A96" :to="step.to">
```

Comment:
I'm not certain whether this works with v-btns, but in HTML if an element doesn't handle an event (eg, click) then it bubbles up to its parent. So the action on this button might not be needed. Or maybe it doesn't need to be a button at all.
(Bubbling is why we stop event propagation on some buttons.)

Reply:
The `ripple` is to stop that default animation that the Vuetify `v-btns` have when being selected. 
Or are you referring to the `:to` property?

Reply:
Yes, I was thinking that this button could "do nothing" and its parent div could handle the bubbled-up event. Not sure if this is possible here, but it's a common technique for divs that work as buttons.

Reply:
Great question, when i remove the `:to` binding, it removes the btns access to its `active-class` binding. I have no idea why or how, but the result is the underline styling that follows the selected step, doesn't work. It will require some restructuring to figure out, since we totally made this custom :) 
Can we defer this?

Reply:
Yes, let's defer this, as it's currently working correct (just not perfect code-wise). It's low priority so I'm not even going to create a ticket for it atm.